regulations/views/comment.py

```diff
@@ -23,6 +31,7 @@ def upload_proxy(request):
     url = s3.generate_presigned_url(
         ClientMethod='put_object',
         Params={
+            'ContentLength': size,
```

Comment:
Done. And I wasn't able to upload files that didn't match the content length in the signature.